omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java

```diff
@@ -90,10 +90,8 @@ public void controller(FragmentConfiguration config,
     	CollectionUtils.filter(ret, new Predicate<Extension>() {
     		@Override
     		public boolean evaluate(Extension extension) {
-       		 	if(extensionPointId.equals(extension.getExtensionPointId())) {
-	              return true;
-	            }
-	            return false;
+       		 	Boolean result = extensionPointId.equals(extension.getExtensionPointId());
+       		 	return result;
```

Comment:
What about simply using (return extensionPointId.equals(extension.getExtensionPointId());

Reply:
@HerbertYiga I thought storing a value in a variable first is easy to debug. Could you advise on the preference for openmrs so that i make changes?

Reply:
use return extensionPointId.equals(extension.getExtensionPointId());

Reply:
@HerbertYiga Changes have been successfully made